contrib/babelfishpg_tsql/src/properties.c

```diff
@@ -143,23 +146,115 @@ Datum connectionproperty(PG_FUNCTION_ARGS) {
 	PG_RETURN_BYTEA_P(convertVarcharToSQLVariantByteA(vch, PG_GET_COLLATION()));
 }
 
-void* get_servername_helper()
+static VarChar *
+get_servername_helper()
 {
-	StringInfoData temp;
-    void* info;
+	StringInfoData	temp;
+	void*		info;
 
     initStringInfo(&temp);
     appendStringInfoString(&temp, bbf_servername);
 
     info = tsql_varchar_input(temp.data, temp.len, -1);
     pfree(temp.data);
-    return info;
+    return (VarChar *)info;
+}
+
+static char *
+ProcessVersionNumber(const char* inputString)
```

Comment:
We can remove unnecessary `GetConfigOption` and strcasecmp(inputString, "product_major_version") calls, e.g.,
```
static char *get_version_number(const char *version_string, int idx)
--> return token when part == idx
```